src/main/java/org/opentripplanner/api/common/RoutingResource.java

```diff
@@ -194,36 +244,47 @@
     protected Integer numItineraries;
 
     /**
-     * The list of preferred routes. The format is agency_[routename][_routeid], so TriMet_100 (100 is route short name)
-     * or Trimet__42 (two underscores, 42 is the route internal ID).
+     * The comma-separated list of preferred agencies.
+     *
+     * @deprecated TODO OTP2: Needs to be implemented
      */
-    @QueryParam("preferredRoutes")
-    protected String preferredRoutes;
-
-    /** Penalty added for using every route that is not preferred if user set any route as preferred, i.e. number of seconds that we are willing
-     * to wait for preferred route. */
-    @QueryParam("otherThanPreferredRoutesPenalty")
-    protected Integer otherThanPreferredRoutesPenalty;
-    
-    /** The comma-separated list of preferred agencies. */
+    @Deprecated
     @QueryParam("preferredAgencies")
     protected String preferredAgencies;
     
+
     /**
-     * The list of unpreferred routes. The format is agency_[routename][_routeid], so TriMet_100 (100 is route short name) or Trimet__42 (two
-     * underscores, 42 is the route internal ID).
+     * The comma-separated list of unpreferred agencies.
+     *
+     * @deprecated TODO OTP2: Needs to be implemented
      */
-    @QueryParam("unpreferredRoutes")
-    protected String unpreferredRoutes;
-    
-    /** The comma-separated list of unpreferred agencies. */
+    @Deprecated
     @QueryParam("unpreferredAgencies")
     protected String unpreferredAgencies;
 
-    /** Whether intermediate stops -- those that the itinerary passes in a vehicle, but 
-     *  does not board or alight at -- should be returned in the response.  For example,
-     *  on a Q train trip from Prospect Park to DeKalb Avenue, whether 7th Avenue and
-     *  Atlantic Avenue should be included. */
+    /**
+     * The comma-separated list of banned agencies.
+     *
+     * @deprecated TODO OTP2 Regression. Not currently working in OTP2.
```

Comment:
How do these types of deprecation comments differ from "TODO OTP2: Needs to be implemented"? Is this something that can be reimplemented in the future but no plans to implement it in the near future?

Reply:
I guess you have a point, the plan is to make sure we go trough all `TODO OTP2` before the release of OTP2, but we will clean up the REST API before that. Also, resolving a TODO might not mean fixing it, it can be just to create a new issue here at GitHub